.devcontainer/license_checker.py

```diff
@@ -52,7 +52,6 @@
     "example-config.yaml",
     "LICENSE",  # is checked but not for the license header
     ".pre-commit-config.yaml",
-    "README.md",
```

Comment:
No license for the README?

Reply:
THe licencs checker is not able to check any .md files. We could still add it, but the checker would not be able to confirm it.

Reply:
Why can't it check? Isn't it just plaintext?

Reply:
It is markdown, which means, that the licence has to be put in as a comment. Somehow, the license-checker is not able to work with markdown comments and thus reports all markdown files. Since the readme-file was excluded before anyways, I switched it to exclude all .md files in general.